src/launch.py

```diff
@@ -25,7 +25,7 @@ def launch(config):
         bpy.context.scene.unit_settings.system = "METRIC"
         bpy.context.scene.unit_settings.scale_length = 1.0  # Set the scale to 1.0 for metric units
         cropcon = CropController(config, collection)
-        scenerender = SceneRenderer(config["outfile"][0], collection)
+        scenerender = SceneRenderer(config["outfile"][0], collection, config["resolution"])
```

Comment:
I think it may be better to just pass `config` as one attribute and then access it later